lib/MusicBrainz/Server/Edit/Work/RelatedEntities.pm

```diff
@@ -13,12 +13,29 @@ around '_build_related_entities' => sub
         $self->c->model('Work')->get_by_ids($self->work_ids)
     };
 
+    my ($recordings, $recording_hits) = $self->c->model('Recording')->find_by_works($self->work_ids);
```

Comment:
(This currently works since `work_ids` just returns one ID, and `find_by_works` accepts that, but assuming the above change)
```suggestion
    my ($recordings, $recording_hits) = $self->c->model('Recording')->find_by_works([$self->work_ids]);
```